cmd/trace-agent/config/config.go

```diff
@@ -597,3 +594,55 @@ func acquireHostnameFallback(c *config.AgentConfig) error {
 	log.Debugf("Acquired hostname from core agent (%s): %q.", c.DDAgentBin, c.Hostname)
 	return nil
 }
+
+//UpdateConfigHandler returns handler for configuration changes during runtime
+func UpdateConfigHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		_ = req.ParseForm()
+
```

Comment:
```suggestion
```

Reply:
I'm assuming this means I should remove an empty line below.